natlas-agent/Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "e278b91636323422e2ed88595982159b19bf80a9ad87826d8153f9b7b6af431a"
+            "sha256": "b0e09cd3f29a45974b789b5971a552ed52a68a27b104785fd1c6d02391bf4c2f"
```

Comment:
Why do so many hashes change when we install one new package?

Reply:
The top hash is a hash of all the packages installed, I think. Then more hashes are changing because versions are getting bumped automatically per the rules in `Pipfile` which mostly say either `"*"` which matches any version, or `~=x.x` which matches any compatible version. We could make this happen less often by restricting to specific versions in Pipfile, though I don't think there's any particular value in that.